File: go/wam/4214.go
const WamChatLockDaily = 4214 // channel: regular

type ChatLockDailyEvent struct {
	FolderChatsCount int64 `wam:"folder_chats_count"` // field 1
	FolderOpenCount int64 `wam:"folder_open_count"` // field 2
	NewAddChatCount int64 `wam:"new_add_chat_count"` // field 3
	NewRemoveChatCount int64 `wam:"new_remove_chat_count"` // field 4
	LockFolderHidden bool `wam:"lock_folder_hidden"` // field 5
	SecretCodeActive bool `wam:"secret_code_active"` // field 6
}

File: rust/wam/4214.rs
pub const WAM_CHAT_LOCK_DAILY: u32 = 4214;

#[derive(Debug, Default)]
pub struct ChatLockDailyEvent {
    /// field 1, wire `folder_chats_count`
    pub folder_chats_count: Option<i64>,
    /// field 2, wire `folder_open_count`
    pub folder_open_count: Option<i64>,
    /// field 3, wire `new_add_chat_count`
    pub new_add_chat_count: Option<i64>,
    /// field 4, wire `new_remove_chat_count`
    pub new_remove_chat_count: Option<i64>,
    /// field 5, wire `lock_folder_hidden`
    pub lock_folder_hidden: Option<bool>,
    /// field 6, wire `secret_code_active`
    pub secret_code_active: Option<bool>,
}